blue-note/theme.json

```diff
@@ -217,6 +217,25 @@
 				"typography": {
 					"fontSize": "var(--wp--preset--font-size--medium)"
 				}
+			},
+			"core/post-title": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--extra-large)"
+				}
+			},
+			"core/post-content": {
+				"typography": {
+					"fontSize": "var(--wp--preset--font-size--normal)",
+					"lineHeight": "1.6"
+				}
+			},
+			"core/image": {
+				"spacing": {
+					"margin": {
+						"top": "6.25rem!important",
```

Comment:
I´m not sure about this because it´s not overrideable by the user since the image block doesn´t have spacing controls